src/App.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
-import Navbar from './components/Navbar';
-import Categories from './components/Categories';
-import RenderBooks from './components/RenderBooks';
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navigation from './components/navigation';
+import BooksDisplay from './components/booksDisplay';
+import Categories from './components/categories';
```

Comment:
- [ ] Based on the requirements of this project, the addBook and removeBook features should still function as before and the app should open in the browser. I got the console error on the screenshot below. Kindly make sure to reference the correct file(Categories not categories)